libs/angular/admin/class-printouts/project.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "angular-admin-class-printouts",
```

Comment:
New libraries are generated with the Nx CLI tool. Files like this are auto-generated.